flask_debugtoolbar/panels/__init__.py

```diff
@@ -25,6 +25,29 @@ def __init__(self, jinja_env, context={}):
         # If the client enabled the panel
         self.is_active = False
 
+    @classmethod
+    def init_app(cls, app):
+        """Method that can be overridden by child classes.
+        Can be used for setting up additional URL-rules/routes.
+
+        Example::
+
+            class UMLDiagramPanel(DebugPanel):
```

Comment:
Probably :)
What specifically are you referring to?

On Tuesday, June 7, 2016, illume <notifications@github.com> wrote:

> In flask_debugtoolbar/panels/__init__.py
> <https://github.com/mgood/flask-debugtoolbar/pull/95#discussion_r66018149>
> :
>
> > @@ -25,6 +25,29 @@ def __init__(self, jinja_env, context={}):
> >          # If the client enabled the panel
> >          self.is_active = False
> >
> > +    @classmethod
> > +    def init_app(cls, app):
> > +        """Method that can be overridden by child classes.
> > +        Can be used for setting up additional URL-rules/routes.
> > +
> > +        Example::
> > +
> > +            class UMLDiagramPanel(DebugPanel):
>
> Typo?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mgood/flask-debugtoolbar/pull/95/files/aaf0ef14bf712bd17b59e182c7d0c2bdd7a861c6#r66018149>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAllYN9iN5lgOTZx43haO5JfmYt5hHYWks5qJRTLgaJpZM4IvAbX>
> .
>